lib/reporters/Html.js

```diff
@@ -128,6 +128,65 @@ define([
 			document.head.appendChild(link);
 		}
 
+		function toggleCollapse(node) {
+			var indented; // keep track of current indent level
```

Comment:
Prefer comments over code rather than after. Also, it's not necessary to comment reasonably obvious things (or things that will become obvious when reading the code).